packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -35,7 +38,23 @@ There are some column sizing behaviors that we will ultimately expect to support
 
 ### API
 
-To accomodate the various sizing modes of a column, in addition to the other necessary behaviors, we can add the following properties to `ColumnInternals`:
+`Table`
```

Comment:
Naming suggestions:
1. stick with `disable` for both properties rather than `restrict`
2. use `resize` instead of `sizing` to match browser APIs like [resize](https://developer.mozilla.org/en-US/docs/Web/CSS/resize). So `disable-column-resize` and `disableColumnResize`
3. Consider alternate names for the table resizing attribute
    - `disable-resize-grow`
    - `disable-column-resize-autogrow`
4. Consider changing them both to enums to allow for future growth and make the API more expressive (also inspired by `resize='none' | 'horizontal' | 'vertical' | 'both'`)
   - `column-resize = 'none' | 'push-pull'`
   - `column-resize-table-width = 'none' | 'grow'
5. Consider combining the two attributes to a single enum.
   - `column-resize= 'none' | 'grow-table' | 'limit-width'`?

Just a starting point for brainstorming. I'm not attached to any of these if they don't cover the behavior accurately.

Reply:
At the moment I'm leaning towards the last suggestion. What about:
-    ```column-resize: 'none' | 'cascade' | 'cascade-limit-width';```